src/routes/web.php

```diff
@@ -1,20 +1,25 @@
 <?php
 
-$url = config('scheduler.url');
-$controller = 'MatviiB\\Scheduler\\Controllers\\SchedulerController';
+Route::group([ 'namespace' => 'MatviiB\\Scheduler\\Controllers',
+    'middleware' => config('scheduler.middleware')], function () {
 
```

Comment:
Let's add the prefix for group as `'prefix' => $url`

Reply:
As you know, prefixing only effects on url paths. But the problem is that when "slash" exists in config('scheduler.url'), we should replace it with "dot".
I have updated SchedulerController to fix this issue.
This is a quick fix, but I'm looking for a better way to name the routes.
Thanks

Reply:
If you have any idea for solving this problem globally, please tell me.
Thanks